mmv1/templates/terraform/examples/bigquery_connection_kms.tf.erb

```diff
@@ -0,0 +1,24 @@
+resource "google_bigquery_connection" "<%= ctx[:primary_resource_id] %>" {
+   connection_id = "<%= ctx[:vars]['connection_id'] %>"
+   location      = "aws-us-east-1"
+   friendly_name = "👋"
+   description   = "a riveting description"
+   kms_key_name  = "<%= ctx[:vars]['kms_key_name'] %>"
+   aws { 
+      access_role {
+         iam_role_id =  "<%= ctx[:vars]['iam_role_id'] %>"
+      }
+   }
+
+  depends_on = [
+    google_kms_crypto_key_iam_member.crypto_key
+  ]
+}
+
+resource "google_kms_crypto_key_iam_member" "crypto_key" {
```

Comment:
Rather than adding this as part of the configuration, use [`BootstrapAllPSARoles`](https://github.com/hashicorp/terraform-provider-google-beta/blob/main/google-beta/acctest/bootstrap_iam_test_utils.go) (or one of its variants) to grant the permissions for this service account one time at the project level - this avoids accidentally removing these permissions if multiple tests try to use the key at the same time.